gateway/gateway.go

```diff
@@ -282,7 +282,7 @@ func (gw *Gateway) handleMessage(msg config.Message, dest *bridge.Bridge) []*BrM
 
 	// Get the ID of the parent message in thread
 	var canonicalParentMsgID string
-	if msg.ParentID != "" && (gw.BridgeValues().General.PreserveThreading || dest.GetBool("PreserveThreading")) {
+	if msg.ParentID != "" && dest.GetBool("PreserveThreading") {
```

Comment:
This removes  `PreserveThreading` from `general`, why?
(can also be used for mattermost when implemented)